tests/globalhelper/reporthelper.go

```diff
@@ -232,3 +232,19 @@ func containsString(list []string, item string) bool {
 
 	return false
 }
+
+func CopyClaimFileToTcFolder(tcName, formattedTcName string) {
```

Comment:
I agree, but this will cause the LaunchTests func to become more unclear about errors, 
in the future, refactoring should be done to this function fixing readability, and then we will apply this.